testsuite/features/build_validation/migration/sle15sp3_ssh_minion_migration.feature

```diff
@@ -7,6 +7,21 @@ Feature: Migrate a SLES 15 SP3 Salt SSH minion to 15 SP4
   Scenario: Log in as admin user
     Given I am authorized for the "Admin" section
 
+  # pristine images for SSH minions use an old version of OS Salt
+  # Product Migration will be possible only if we update to latest
+  Scenario: Prerequisite: update OS Salt to the latest version
+    Given I am on the Systems overview page of this "sle15sp3_ssh_minion"
+    When I follow "Software" in the content area
+    And I follow "Packages"
+    And I follow "Upgrade"
+    And I enter "salt" as the filtered package name
```

Comment:
I wonder if this filter could be selecting not only the classic salt, but also the salt bundle, as both packages contains the word `salt`.